Makefile

```diff
@@ -38,82 +38,103 @@ DIFF = $(shell git diff --quiet >/dev/null 2>&1; if [ $$? -eq 1 ]; then echo "1"
 ifeq ($(DIFF), 1)
     GIT_TREESTATE = "dirty"
 endif
+PLATFORMS=darwin linux windows
+ARCHITECTURES=amd64
 
 PKG=github.com/sigstore/cosign/pkg/version
+LDFLAGS=-X $(PKG).GitVersion=$(GIT_VERSION) -X $(PKG).gitCommit=$(GIT_HASH) -X $(PKG).gitTreeState=$(GIT_TREESTATE) -X $(PKG).buildDate=$(BUILD_DATE)
 
-LDFLAGS="-X $(PKG).GitVersion=$(GIT_VERSION) -X $(PKG).gitCommit=$(GIT_HASH) -X $(PKG).gitTreeState=$(GIT_TREESTATE) -X $(PKG).buildDate=$(BUILD_DATE)"
-
-.PHONY: all lint test clean cosign cross
+SRCS = $(shell find cmd -iname "*.go") $(shell find pkg -iname "*.go")
 
-all: cosign
+GOLANGCI_LINT_DIR = $(shell pwd)/bin
+GOLANGCI_LINT_BIN = $(GOLANGCI_LINT_DIR)/golangci-lint
 
-help: # Display help
-	@awk -F ':|##' \
-		'/^[^\t].+?:.*?##/ {\
-			printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
-		}' $(MAKEFILE_LIST) | sort
+KO_PREFIX ?= gcr.io/projectsigstore
+export KO_DOCKER_REPO=$(KO_PREFIX)
 
-SRCS = $(shell find cmd -iname "*.go") $(shell find pkg -iname "*.go")
+.PHONY: all lint test clean cosign cross
+all: cosign
 
 cosign: $(SRCS)
-	CGO_ENABLED=0 go build -trimpath -ldflags $(LDFLAGS) -o $@ ./cmd/cosign
+	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" -o $@ ./cmd/cosign
 
 cosign-pivkey: $(SRCS)
 	CGO_ENABLED=1 go build -trimpath -tags=pivkey -ldflags $(LDFLAGS) -o cosign ./cmd/cosign
 
 cosign-pkcs11key: $(SRCS)
 	CGO_ENABLED=1 go build -trimpath -tags=pkcs11key -ldflags $(LDFLAGS) -o cosign ./cmd/cosign
```

Comment:
double quotes are missing here too.

Reply:
thanks for the feedback and review Hector!

Reply:
I'm always around when I have some free time 😄 .